src/Plan/Plan.php

```diff
@@ -111,6 +119,25 @@ public function setName(string $name)
         return $this;
     }
 
+    /**
+     * @return array
+     */
+    public function paymentMethod()
+    {
+        return $this->paymentMethod ?? $this->firstPaymentMethod;
```

Comment:
These variables are two slightly different things, need to double check this.

Reply:
If the plan doesn't have defined payment method, plas use as paymentMethod the firstPaymentMethod defined.

Reply:
Check in last commit... now I understand